pkg/gui/wallet_loader.go

```diff
@@ -0,0 +1,79 @@
+package gui
+
+import (
+	"errors"
+
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/container"
+	"fyne.io/fyne/v2/layout"
+	"fyne.io/fyne/v2/widget"
+)
+
+type WalletInfoEntry struct {
+	ClearPassword string
+	WalletName    string
+	PrivateKey    string
+	Err           error
+}
+
+func AskWalletInfo(app *fyne.App) (string, string, string, error) {
+	return WalletInfo(app)
+}
```

Comment:
this function looks useless as we can call directly `WalletInfo`

Reply:
@Hugo ?

Reply:
I agree but that's how it is done in other components of the GUI so either we change everything or we keep it like this but we must keep consistency.